cg/meta/workflow/fastq.py

```diff
@@ -254,6 +254,25 @@ def create_fastq_name(
 
         return f"{flowcell}_L{lane}_{meta}_{read}.fastq.gz"
 
+
+class RnafusionFastqHandler(FastqHandler):
+    @staticmethod
+    def create_fastq_name(
+        lane: str,
+        flowcell: str,
```

Comment:
I am not against the change, but that would make the spelling inconsistent between rnafusion and the other pipelines

Reply:
You have to start somewhere 😄. Other approach is to make one or several PRs to fix only that, I think it is nice to clean as you go along and you will be testing this PR now. But, It can also wait